### PR TITLE
Deploy local registry and router in a single playbook

### PR DESCRIPTION
**Pull request by maxamillion**
(use azure blob storage on the back end)

Signed-off-by: Adam Miller <admiller@redhat.com>

### CODE REVIEW

**Comment by maxamillion** [comment]
cc @jim-minter

**Comment by jim-minter** [comment]
also can you remove the lines " # TODO: when enabling secure registry, may need:
 # ln -s /etc/origin/node/node-client-ca.crt /etc/docker/certs.d/docker-registry.default.svc:5000"

File: pkg/openshift/certgen/templates/master/tmp/ansible/ansible.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -x
 
-# TODO: do this, and more (registry console, asb), the proper way
```
[the remark this answers is not in the document]
Comment (maxamillion) [reply pkg/openshift/certgen/templates/master/tmp/ansible/ansible.sh]: @jim-minter ah, I thought it was missing a comma and was talking about the registry and the web console

**Pull request merged**